python/MetrologyData.py

```diff
@@ -112,7 +111,21 @@ def set_ref_plane(self, plane_functor, zoffset=0, nsigma=5):
         pos, z = self.sensor.data()
         dz = z - plane_functor(pos) + zoffset
         self.resids = dz
+
         # Also define residuals with outliers removed (nsigma clipping)
+        if self.sensor.mean_filt == None:
```

Comment:
Thanks, Jim.  I've implemented this change and your recommendations below in the updates that I just committed.